modules/order/src/Element/ProfileSelect.php

```diff
@@ -100,6 +183,85 @@ public static function processForm(array $element, FormStateInterface $form_stat
       }
     }
 
+    $called_class = get_called_class();
+    if (!empty($profile_uid) && $mode != 'edit' && !empty($profile_options)) {
+      $element['profile_selection'] = [
+        '#title' => t('Select an address'),
+        '#options' => $profile_options,
+        '#type' => 'select',
+        '#weight' => -5,
+        '#default_value' => $default_profile_id,
```

Comment:
$default_profile_id is not defined here if the user doesn't have any profiles created. Setting it to NULL initially in this function in case it doesn't get redefined seems to work fine for me.